src/NintendoSDK/gfx/detail/gfx_NvnHelper.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include <nn/gfx/gfx_SwapChainInfo.h>
 #include <nn/gfx/gfx_TextureInfo.h>
-#include <nvn/nvn_FuncPtrImpl.h>
+#include <nvn/nvn_FuncPtrBase.h>
```

Comment:
Which one is correct here, should it actually include the `Impl`? Or should I change it back, and you will take care of it in the next PR?

Reply:
It should be fine, it would've been changed in the next PR anyways